### PR TITLE
workspacer: Add version 0.9.9

### PR DESCRIPTION
**Pull request by sitiom**
a tiling window manager for Windows 10

### CODE REVIEW

**Comment by sitiom** [comment]
Oops

File: bucket/workspacer.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+    "version": "0.9.9",
+    "description": "a tiling window manager for Windows 10",
+    "homepage": "https://www.workspacer.org/",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://workspacer.blob.core.windows.net/installers/stable/workspacer-stable-0.9.9.msi"
```
Comment (earnestma) [inline bucket/workspacer.json:8]:
```suggestion
            "url": "https://workspacer.blob.core.windows.net/installers/stable/workspacer-stable-0.9.9.msi"
            "hash": "647b171fa8420ccd07e997a50d440cdbdcdb1cd3472fe50dda8a120ae70cd58d",
```

Comment (sitiom) [reply bucket/workspacer.json]: Fixed.

**Comment by earnestma** [comment]
Opps, there should be no `,` after the end of hash. Sorry about that.

File: bucket/workspacer.json (hunk 1 above)
Comment (earnestma) [reply bucket/workspacer.json]: @sitiom Opps, there should be no , after the end of hash. Could you remove that? Sorry about that.

File: bucket/workspacer.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    "homepage": "https://www.workspacer.org/",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://workspacer.blob.core.windows.net/installers/stable/workspacer-stable-0.9.9.msi",
+            "hash": "647b171fa8420ccd07e997a50d440cdbdcdb1cd3472fe50dda8a120ae70cd58d"
+        }
+    },
+    "extract_dir": "workspacer",
+    "bin": "workspacer.exe",
```
Comment (Ash258) [inline bucket/workspacer.json:13]: There is no command line handling

```suggestion
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Pull request merged**